### DISCUSSION TITLE
Removing hooks from report

### DISCUSSION THREAD

**Discussion by Yevhenii-TA** [mmisty/cypress-allure-adapter#205] [Q&A] [answered]
Hello @mmisty and community. I'm new in Cypress and wanted to ask if there is possibility to remove hooks from test results?
![image](https://github.com/user-attachments/assets/592e610b-6fd8-451b-8fb8-94a02d86bed1)
Thanks in advance

**Comment by mmisty** [comment, marked as answer]
Hello @Yevhenii-TA 

There is environment variable 'allureSkipSteps' where you can add steps that you want to skip.

Read more here https://github.com/mmisty/cypress-allure-adapter?tab=readme-ov-file#environment-variables, row with allureSkipSteps

**Reply by Yevhenii-TA** [reply]
Thanks @mmisty , works perfect!

**Comment by Yevhenii-TA** [comment]
@mmisty , probably this could be related for discussion so will leave it here
I have next message in report
![Screenshot 2024-07-24 at 12 44 55](https://github.com/user-attachments/assets/61ba8825-fb22-4b03-b8ad-f45b5fea6667)
and adding `cypress-allure-adapter` didn't help
I tried to add it in `allureSkipSteps: '"after each" hook*,"before each" hook*, "after all" hook*, "*\\[cypress-allure-adapter\\]*"'` in different ways `cypress-allure-adapter`, `[cypress-allure-adapter]`, '*\\[cypress-allure-adapter\\]*' ...
so don't know a reason why I see this
![image](https://github.com/user-attachments/assets/5e3d5e06-7553-4c20-a867-8dc0701c5e53)

Thanks

**Reply by mmisty** [reply]
thank you, this is defect
Will fix it
